restli-tools/src/main/java/com/linkedin/restli/tools/symbol/RestLiSymbolTableProvider.java

```diff
@@ -163,6 +165,7 @@ public RestLiSymbolTableProvider(Client client,
   {
     _client = client;
     _uriPrefix = uriPrefix;
+    _serverNodeUri = serverNodeUri;
```

Comment:
Should we also check if _serverNodeUri is null before setting _defaultResponseSymbolTable below? 
If we want to allow symbol tables generated by another server, we should still check if that symbol table's nodeUri is not null to be safe. wdyt?

Reply:
Good idea. better safe than sorry.